models/LicenseMember.v1.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "LicenseMember",
+  "description": "Utilisateur attaché à la license Toaztr",
+  "type": "object",
+  "properties": {
+    "ref": {
+      "type": "string",
+      "description": "ID de l'utilisateur."
+    },
+    "name": {
+      "type": "string",
+      "description": "Nom de l'utilisateur."
```

Comment:
On rajoute le mail ?

Reply:
++

Reply:
This is just reflecting what is in SubscriptionItems which do not include the email.
Why should we store the mail here? I don't see the usecase